Further cleanups and fixes for dashboard styles.

solarissmoke opened the pull request:
- Remove unnecessary CSS in favour of Bootstrap defaults (also remove some completely unused CSS).
- Remove use of `btn-lg` in the dashboard, whose presentation was being overridden in CSS to make it look like a normal button anyway.
- Fix a few classes on HTML elements.

jwayodi commented:
I think you can also remove the rules below from `src/oscar/static_src/oscar/scss/dashboard/base.scss`:
```
.bs-docs-sidenav .nav-link.active {
  color: $link-color;
  &:hover {
    background: $gray-200;
    color: $link-color;
  }
}
```

jwayodi commented:
You can also remove the `.btn-lg` in `src/oscar/templates/oscar/dashboard/login.html`?

jwayodi commented on src/oscar/templates/oscar/dashboard/offers/progress.html:
```diff
@@ -11,9 +11,9 @@ <h3>{% trans "Steps" %}</h3>
             </a>
             {% else %}
-            <span class="{% if step == 1 %}active{% else %}disabled{% endif %}">
+            <a class="nav-link {% if step == 1 %}active{% else %}disabled{% endif %}">
```
Maybe you can use `.text-reset` with `.disabled`, so that it does not look like a link?

solarissmoke replied:
Yep, done.

solarissmoke commented:
@jwayodi I've addressed your comments. I had left the btn-lg on the login form deliberately... seemed like a bigger button in that context was not necessarily a bad thing :smile:

jwayodi reviewed (approved):
This looks good.
